packages/flutter/test/material/dropdown_menu_test.dart

```diff
@@ -2129,6 +2129,59 @@ void main() {
     // No exception should be thrown.
     expect(tester.takeException(), isNull);
   });
+
+  // This is a regression test for https://github.com/flutter/flutter/issues/147076.
+  testWidgets('Text field does not overflow parent', (WidgetTester tester) async {
+    await tester.pumpWidget(MaterialApp(
+      home: Scaffold(
+        body: SizedBox(
+          width: 300,
+          child: DropdownMenu<int>(
+            dropdownMenuEntries: <DropdownMenuEntry<int>>[
+              DropdownMenuEntry<int>(
+                value: 0,
+                label: 'This is a long text that is multiplied by 4 so it can overflow. ' * 4,
+              ),
+            ],
+          ),
+        ),
+      ),
+    ));
+
+    await tester.pump();
+    final RenderBox box = tester.firstRenderObject(find.byType(TextField));
+    expect(box.size.width, 300.0);
+  });
+
+  // This is a regression test for https://github.com/flutter/flutter/issues/147173.
+  testWidgets('Text field with large helper text can be selected', (WidgetTester tester) async {
+    await tester.pumpWidget(
+     const MaterialApp(
```

Comment:
```suggestion
    await tester.pumpWidget(const MaterialApp(
```
Similarly to the other tests in this file.

Reply:
Thanks for the suggestion. I've fixed the above issues.